spec/features/user_browses_loan_requests_spec.rb

```diff
@@ -1,22 +1,32 @@
 require "rails_helper"
 
-RSpec.feature "user browses items" do
-  let(:create_item) { LoanRequest.create( ) }
-
-  context "unauthenticated user" do
-    xscenario "can browse the items" do
-      # create(:item)
-      visit "/menu"
-      expect(current_path).to eq(menu_path)
-      expect(page).to have_content("Sushi")
+RSpec.feature "unauthenticated user browses loan requests" do
+  let(:create_item) { LoanRequest.create(title: "Farm Tools",
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.